Jenkinsfile

```diff
@@ -41,4 +41,4 @@
 //@Library(value="pipeline-lib@your_branch") _
 
 // centos8: No matching package to install: 'fio-devel'
-packageBuildingPipelineDAOS(['distros': ['centos7', 'el8', 'leap15', 'ubuntu20.04']])
+packageBuildingPipelineDAOS(['distros': ['centos7', 'el8', 'leap15']])
```

Comment:
Can you update the comment here (remove the comment about centos8 since it's clearly out of date) to document why Ubuntu is being removed please?  Include the DAOS-10291 reference in the comment.

Reply:
done

Reply:
Ubuntu not being built because of an issue with the common packaging makefile not being able to deal with "patch10" as expected.  It is having ubuntu apply it before "patch2", so the Ubuntu build fails.

Reply:
I squashed two of the patches into one so that there are only 9 to avoid the DAOS-10291. This enabled me to add back the ubuntu20.04 as otherwise I'm concerned it may affect CI results in general if landed with ubuntu disabled.